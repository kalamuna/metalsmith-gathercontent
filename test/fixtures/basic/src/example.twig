<div>{{gatherContent.home.name}}</div>
