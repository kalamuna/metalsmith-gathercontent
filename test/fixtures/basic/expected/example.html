<article id="page-4416626">
    <main class="about-us" data-id="4416626">
        <h1 class="about-us--title">About Us</h1>
    </main>
</article>